gravitee-am-management-api/gravitee-am-management-api-services/gravitee-am-management-api-services-purge/src/main/java/io/gravitee/am/management/services/purge/ScheduledPurgeService.java

```diff
@@ -45,17 +46,17 @@ public class ScheduledPurgeService extends AbstractService implements Runnable {
     @Autowired
     private TaskScheduler scheduler;
 
-    @Value("${services.purge.cron:0 0 23 * * *}") // execute every day at 11 PM
-    private String cronTrigger;
+    @Value("${services.purge.cron}")
+    private String cronTrigger = "0 0 23 * * *"; // execute every day at 11 PM
 
```

Comment:
On missing purge.cron configuration value, the application won't get up. Is it expected?

Reply:
fixed